From 12098b25406804372e3488b0560e8896addff7b6 Mon Sep 17 00:00:00 2001
From: Caesar Wang <wxt@rock-chips.com>
Date: Tue, 10 Nov 2020 15:52:16 +0800
Subject: [PATCH] arm64/configs: rockchip_linux_defconfig: enable
 CONFIG_DRM_IGNORE_IOTCL_PERMIT

Allow the DRM ignore some permission issues.

Signed-off-by: Caesar Wang <wxt@rock-chips.com>
Change-Id: Ia4c80fec7fa0757f5797ee111f96c9f1d41c9277
---
 arch/arm64/configs/rockchip_linux_defconfig | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm64/configs/rockchip_linux_defconfig b/arch/arm64/configs/rockchip_linux_defconfig
index e43b073b4962..35e10a9c1203 100644
--- a/arch/arm64/configs/rockchip_linux_defconfig
+++ b/arch/arm64/configs/rockchip_linux_defconfig
@@ -286,6 +286,7 @@ CONFIG_VIDEO_OV4689=y
 CONFIG_VIDEO_OV13850=y
 # CONFIG_VGA_ARB is not set
 CONFIG_DRM=y
+CONFIG_DRM_IGNORE_IOTCL_PERMIT=y
 CONFIG_DRM_LOAD_EDID_FIRMWARE=y
 CONFIG_DRM_ROCKCHIP=y
 CONFIG_ROCKCHIP_ANALOGIX_DP=y
-- 
2.35.3

